features/account/registering.feature

```diff
@@ -17,6 +18,7 @@ Feature: Account registration
         And I confirm this password
         And I register this account
         Then I should be notified that new account has been successfully created
+        And I should be on registration thank you page
```

Comment:
I know that we can not fix it quickly but this step has no sense in API context :(

Reply:
Maybe create a separate scenario for API context here (without this step and with some steps merged)?